test/cli/test_catalog.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+from unittest.mock import patch, MagicMock
+from click.testing import CliRunner
+from ersilia.cli.commands.catalog import catalog_cmd
+
+@pytest.fixture
+def runner():
+    return CliRunner()
+
+@patch("ersilia.hub.content.catalog.ModelCatalog.airtable", return_value=None)
+@patch("ersilia.hub.content.catalog.ModelCatalog.local")
```

Comment:
We shouldn't be mocking this because it's not an external interface, but information available on the disk.

Reply:
Solved, now local can read models that are written in `eos` folder. Nice catch @DhanshreeA.